creation/singleton-classic/README.md

```diff
@@ -0,0 +1,15 @@
+# Singleton Classic
+
+Using getInstance static method instead of default object constructor allows us to skip re-instantiation and return the previously created instance. Note that we call this method in three different locations in the code; however, each getInstance call returned same instance. So counter could continue with previous state (history). We were able to count till seven across different files and functions.
```

Comment:
Typo: It is counted till ten but written seven.

Below I suggested a corrected version with some grammar corrections as well.

Using the getInstance static method instead of the default object constructor allows us to skip re-instantiation and return the previously created instance. Note that we call this method in three different locations in the code; however, each getInstance call returned the same instance. So counter could continue with the previous state (history). We were able to count to ten across different files and functions.

Reply:
Updated